ai_edge_torch/generative/layers/unet/blocks_2d.py

```diff
@@ -147,7 +154,11 @@ def __init__(self, config: unet_cfg.CrossAttentionBlock2DConfig):
     self.config = config
     self.norm = layers_builder.build_norm(config.query_dim, config.normalization_config)
     self.attention = CrossAttention(
-        config.query_dim, config.cross_dim, config.attention_config, 0, True
+        config.query_dim,
+        config.cross_dim,
+        config.attention_config,
+        0,
+        enable_hlfb=True,
```

Comment:
can you read this `enable_hlfb` from Model config class instead of setting to True always?

Reply:
done